### PR TITLE
[docs] Revise README.md

### PR DESCRIPTION
**Pull request by Mudassar045**
Basically we don't need the `return` inside the `switch` statement, it will jump us out of the circuit.

### CODE REVIEW

**Comment by Mudassar045** [comment]
> @googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ const MyComponent = () => {
 
 #### Code-loading
 
-Deliver a light, interactive core experience to users and progressively add high-end-only features on top, if a users hardware can handle it. Below is an example using the Network Status hook:
+Deliver a light, interactive core experience to users and progressively add high-end-only features on top, if a user's hardware can handle it. Below is an example using the Network Status hook:
```
Comment (thomasbritton) [inline README.md]: I don't think this should have an apostrophe added here as it's a plural

Comment (Mudassar045) [reply README.md]: `if a users hardware`, must be changed to `if users' hardware` or `if a user's`

**Comment by addyosmani** [comment]
Thank you, @Mudassar045!

**Pull request merged**